app/uk/gov/hmrc/taxenrolmentassignmentfrontend/orchestrators/MultipleAccountsOrchestrator.scala

```diff
@@ -25,24 +25,37 @@ import uk.gov.hmrc.service.TEAFResult
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.AccountTypes
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.AccountTypes.{
   MULTIPLE_ACCOUNTS,
-  SA_ASSIGNED_TO_CURRENT_USER
+  SA_ASSIGNED_TO_CURRENT_USER,
+  SA_ASSIGNED_TO_OTHER_USER
 }
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.auth.RequestWithUserDetails
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.errors.InvalidUserType
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.AccountDetails
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.errors.{
+  InvalidUserType,
+  TaxEnrolmentAssignmentErrors
+}
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.{
+  AccountDetails,
+  UsersAssignedEnrolment
+}
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.repository.SessionKeys.{
   ACCOUNT_TYPE,
-  REDIRECT_URL
+  REDIRECT_URL,
+  REPORTED_FRAUD,
+  USER_ASSIGNED_SA_ENROLMENT
 }
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.repository.TEASessionCache
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.services.UsersGroupSearchService
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.services.{
+  SilentAssignmentService,
+  UsersGroupSearchService
+}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class MultipleAccountsOrchestrator @Inject()(
   sessionCache: TEASessionCache,
-  usersGroupSearchService: UsersGroupSearchService
+  usersGroupSearchService: UsersGroupSearchService,
+  silentAssignmentService: SilentAssignmentService
 ) {
 
   def getDetailsForLandingPage(
```

Comment:
Just for consistency can you change this from LandingPage to NotificationPage, ty!